src/BlazorWebView/src/Wpf/BlazorWebView.cs

```diff
@@ -32,6 +32,14 @@ public class BlazorWebView : Control, IAsyncDisposable
 			ownerType: typeof(BlazorWebView),
 			typeMetadata: new PropertyMetadata(OnHostPagePropertyChanged));
 
+		/// <summary>
+		/// The backing store for the <see cref="StartPath"/> property.
+		/// </summary>
+		public static readonly DependencyProperty StartPathProperty = DependencyProperty.Register(
+			name: nameof(StartPath),
+			propertyType: typeof(string),
+			ownerType: typeof(BlazorWebView));
```

Comment:
It seems that in WPF you can set the default value in the dependency property like this:

```suggestion
		public static readonly DependencyProperty StartPathProperty = DependencyProperty.Register(
			name: nameof(StartPath),
			propertyType: typeof(string),
			ownerType: typeof(BlazorWebView),
			typeMetadata: new PropertyMetadata("/"));
```

And then you can delete the line below that sets it in the ctor.

But please try it.